concourse/tasks/run-smoke-tests.yml

```diff
@@ -0,0 +1,15 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: ((readonly_private_ecr_repo_url))
+    tag: govuk-coronavirus-find-support-feature-tests
+params:
+  TEST_URL:
+  RAILS_ENV: smoke_test
```

Comment:
One reason to have the `RAILS_ENV` be `smoke_test` rather than `test` was because we didn't want to setup a test db locally in the docker container. I wonder if this is necessary here, since in this commit it looks like active record is still loaded. Does running `TEST_URL=x RAILS_ENV=smoke_test bundle exec rspec spec/features` locally work when the DB is not available?

Reply:
@bilbof ah yeah... it does not as the only available databases configurations are: `default, development, test, production`. 

Was the reason for not wanting to set up the test db in the docker container more to do with needing to get something up and running quickly, or was it to do with not wanting to persist the data/needing to identify that the data is from the smoke test to then remove it (for prod)?

If it is ok to persist, then I guess we can remove the `smoke_test` rails env and use `test`... though our docker-image for `run-smoke-tests`  would need to support chrome and postgres... so I guess we could do something to `run-quality-checks` and load that same Dockerfile but just run `bundle exec rspec spec/features` with the `test_url` as opposed to `rake`? May be good to catch up about this on a quick call?

Reply:
Yeah, iirc when we set this up we didn't have the docker container. Now that we do, perhaps we could switch to using the test env. We shouldn't need a DB connection to run the smoke tests, but it won't hurt.

>  could do something similar to run-quality-checks and load that same Dockerfile but just run bundle exec rspec spec/features with the test_url as opposed to rake

Yeah I think that makes sense, I'd go for that. I'm available for a call if you want to discuss it.

Reply:
added a commit to use the header to check if smoke test.